Add SAUCE_CONNECT_OVERRIDES_PATH (#75)
DevExpress/testcafe-browser-provider-saucelabs: README.md

 - `SAUCE_BUILD` - the text that will be displayed as Build Name on SauceLabs.

 - `SAUCE_CONFIG_PATH` - path to a file which contains additional **job options** as JSON. See [SauceLabs Test Configuration](https://wiki.saucelabs.com/display/DOCS/Test+Configuration+Options#TestConfigurationOptions-TestAnnotation) for a full list.

>  - `SAUCE_CONNECT_OVERRIDES_PATH` = path to a file which contains overrides for the the connector. See [Sauce Connect Launcher Advanced Usage](https://github.com/bermi/sauce-connect-launcher#advanced-usage) for details.

Reviewer: @titerman, could you please check this description? I suggest something like the following:
```suggestion
 - `SAUCE_CONNECT_OVERRIDES_PATH` - path to a file which contains overrides for the Sauce Connect options to be passed to the [SauceLabs connector](https://github.com/DevExpress/saucelabs-connector). See [Sauce Connect Launcher Advanced Usage](https://github.com/bermi/sauce-connect-launcher#advanced-usage) for details about these options.
```

Author: I went ahead and made the update. Let me know if there is anything else I can do. Happy to help.

>  - `SAUCE_CONNECT_OVERRIDES_PATH` - path to a file which contains overrides for the Sauce Connect options to be passed to the [SauceLabs connector](https://github.com/DevExpress/saucelabs-connector). See [Sauce Connect Launcher Advanced Usage](https://github.com/bermi/sauce-connect-launcher#advanced-usage) for details about these options.

Reviewer 2: ```suggestion
 - `SAUCE_CONNECT_OVERRIDES_PATH` - path to a file that overrides SauceLabs [connector options](https://github.com/DevExpress/saucelabs-connector). See [Sauce Connect launcher documentation](https://github.com/bermi/sauce-connect-launcher#advanced-usage) for more information.
```
